build_tools/docker/README.md

```diff
@@ -69,38 +69,95 @@ python3 build_tools/docker/manage_images.py --images all --tag latest --update_r
 This requires that the tagged image have a repository digest, which means it was
 pushed to or pulled from GCR.
 
-## Deploying New Images
+## Adding or Updating an Image
 
-1.  Modify the Dockerfiles as desired.
-2.  Update `manage_images.py` to include the new image and its dependencies.
-3.  Build and push the new image to GCR and update references to it:
+If you have worked with the `docker` images before, it is prudent to follow the
+steps in the "Debugging" section below before continuing.
 
-    ```shell
-    python3 build_tools/docker/manage_images.py --image "${IMAGE?}" --build --push --update_references
+### Part 1. Local Changes
+
+1. Update the `Dockerfile` for the image that you want to modify or add.
+2. Build the image, push the image to GCR and update all references to the image
```

Comment:
This removes the bit about adding an image to `manage_images.py`. I think that's important

Reply:
Done :)